Resources/Prototypes/_Lust/Entities/Mobs/animals.yml

```
@@ -0,0 +1,57 @@
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: _Lust/Mobs/Animals/nine_tailed_fox.rsi
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 50 #They actually are pretty light, I looked it up
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)